AnalysisStep/src/LeptonSFHelper.cc

```diff
@@ -6,6 +6,55 @@ using namespace std;
 
 LeptonSFHelper::LeptonSFHelper(bool preVFP)
```

Comment:
I think that keeping "bool preVPF" is too confusing and limited. Can we change this to pass directly data_tag so that the module handles the different cases for all eras?
It would just need changing also the call from AnalysisStep to work consistently.